spec/defines/dns_logging_category_spec.rb

```diff
@@ -3,7 +3,7 @@
 describe 'dns::logging::category' do
   on_supported_os.each do |os, os_facts|
     context "on #{os}" do
-      let(:facts) { os_facts.merge(fqdn: 'puppetmaster.example.com') }
+      let(:facts) { os_facts.merge(networking: { fqdn: 'puppetmaster.example.com' }) }
```

Comment:
Please use [`override_facts` from voxpupuli-test](https://github.com/voxpupuli/voxpupuli-test#fact-handling) for this (otherwise there are no other networking facts):
```suggestion
      let(:facts) { override_facts(os_facts, networking: { fqdn: 'puppetmaster.example.com' }) }
```

Reply:
Nice!